operator.yaml

```diff
@@ -20,7 +20,7 @@ spec:
       annotations:
         productID: hazelcast-operator
         productName: Hazelcast Operator
-        productVersion: 0.2.0
+        productVersion: 0.2.1
```

Comment:
isn't 0.2.3 the latest one?

Reply:
no, it's `0.2.1` on Docker Hub.

https://hub.docker.com/layers/hazelcast/hazelcast-operator/0.2.1/images/sha256-784e07a4b31b0c296aedf5ee0da423c539ef322c8ea02d23379008b3b40059b0?context=explore